compiler/src/iree/compiler/Dialect/LinalgExt/Utils/WinogradConstants.h

```diff
@@ -45,6 +45,12 @@ const float B_6x6_3x3[] = {
          0.0f,        0.0f,        0.0f,       0.0f,       0.0f,       0.0f,       0.0f,        1.0f
 };
 
+const float GT_6x6_3x3[] = {
```

Comment:
These seem separate to adding the op itself, but ok to slip it in here.

Reply:
Oh oops, this is for the decomposition. I'll move this later